frontend/src/routes/Applications/Overview.tsx

```diff
@@ -699,32 +697,32 @@ export default function ApplicationsOverview() {
         options: [
           {
             label: t('Application set'),
-            value: `${getApiVersionResourceGroup(ApplicationSetApiVersion)}/${ApplicationSetKind}`,
+            value: 'appset',
```

Comment:
@fxiang1 @chenz4027 Because these values will now appear in the browser location bar as the user applies filters, I wanted to make them more human-readable. Did I miss updating them anywhere, or do you see any issues with this? Obviously saved filters will be lost on upgrade to this new format, but that's a one time thing that I'm not concerned about.

Reply:
I think it should be okay  to just use 'appset'

Reply:
Yep should be fine to change the values.